release-team/role-handbooks/ci-signal/README.md

```diff
@@ -77,6 +83,7 @@ Here are some good early deliverables from the CI Signal lead between start of t
     - For failing tests: `[Failing Test]: testgrid-tab-name (prow-job-name)`
     - For flaky tests: `[Flaky Test]: testgrid-tab-name (prow-job-name)`
   - Make sure all open issues have a `priority/` label (see: [Priority Labels](#priority-labels)) and one of either the `kind/flake` or `kind/failing-test` label
+  - cc. @ci-signal github team on the issue to let rest of the team aware of it
```

Comment:
Very minor suggestion but below we say "**CC** the release manager..." and here we say "**cc.** @ci-signal github team...". I don't think it is a huge thing, but just trying to think if someone was very new to k8s / github interactions, it could feel as though these were two different types of actions. What would you think of making them either both *cc.* or both *CC*?

Reply:
Absolutely agree. But you actually got me thinking - who is the release manager? Is that the current release lead? 🤔 or is that this group of people? https://github.com/kubernetes/sig-release/blob/master/release-managers.md what would be the case where you want to ping them? 🤔 almost wondering if we can drop it or change it in some way

Reply:
Great point! I feel like it should maybe be release team lead?

Reply:
Okay, updated. Thanks